cdhweb/people/views.py

```diff
@@ -163,12 +163,12 @@ def get_past_profiles(self):
                 Case(
                     When(user__membership__end_date__isnull=False,
                          then=Max('user__membership__end_date')),
-                    default=Value('0')
+                    default=Value('1900-01-01')
```

Comment:
aha, nice

Reply:
@kmcelwee's idea! sometimes the simplest solution is best, and it works on mysql too, which is nice.